docs/tenancy/1.x/database-mysql.md

```diff
@@ -24,12 +24,14 @@ The purpose of this package is to enable the management (Creation, Updating, Del
 
 **Requirements**
 
-When using the [`hooks-database`](hooks-database) package the MySQL user defined in `database.php` must have the ability to create users, databases, and grant privileges.
+- MySQL server with a version >= `5.7.6`
+- When using the [`hooks-database`](hooks-database) package the MySQL user defined in `database.php` must have the ability to create users, databases, and grant privileges.
+- This package also required the installation and configuration of the [`affects-connections`](https://tenancy.dev/docs/tenancy/1.x/affects-connections) package. 
+  (*Note:* This package is not required for the use of the [`affects-connections`](affects-connections) package)
 
 **Recommendations**
 
 - To automate the management of Tenant Databases when a Tenant is created, updated, or deleted; The [`hooks-database`](hooks-database) package should also be installed.
-- To use the Tenant Database with the `onTenant` trait; The [`affects-connections`](https://tenancy.dev/docs/tenancy/1.x/affects-connections) package should also be installed. (*Note:* This package is not required for the use of the [`affects-connections`](affects-connections) package)
```

Comment:
Why was this removed?

Reply:
Because its *required* now; so the recommendation is a mute point.